pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -341,24 +342,26 @@ def is_optimized_sync(repository, remote, url):
     return is_optimized
 
 
-def synchronize(remote_pk, repository_pk, mirror, skip_types, optimize):
+def synchronize(remote_pk, repository_pk, sync_policy, skip_types, optimize):
     """
     Sync content from the remote repository.
 
     Create a new version of the repository that is synchronized with the remote.
 
-    If mirror=True, a publication will be created with a copy of the original metadata.
-    In this event, SRPMs and other types listed in "skip_types" will *not* be skipped.
+    If sync_policy=mirror_complete, a publication will be created with a copy of the original
+    metadata. This comes with some limitations, namely:
 
-    If mirror=True and the repository uses the xml:base / location_base feature, then
-    the sync will fail. This feature is incompatible with the intentions of most Pulp
-    users, as it will tell clients to look for metadata / packages from a source outside
-    of the repository.
+    * SRPMs and other types listed in "skip_types" will *not* be skipped.
```

Comment:
I think it's worthy to actually add this info to the docs part that clarifies my question in the beginning.